builder/virtualbox/common/step_attach_isos.go

```diff
@@ -0,0 +1,158 @@
+package common
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"path/filepath"
+
+	"github.com/hashicorp/packer/helper/multistep"
+	"github.com/hashicorp/packer/packer"
+)
+
+// This step attaches the boot ISO, cd_files iso, and guest additions to the
+// virtual machine, if present.
+type StepAttachISOs struct {
+	AttachBootIso           bool
```

Comment:
Nit, because it's an abbreviation : 
```suggestion
	AttachBootISO           bool
```